<launch>

  <!-- launch video stream -->
  <include file="$(find opencv_apps)/launch/camshift.launch">
    <arg name="node_name" default="featurization_left" />
    <arg name="image" value="/stereo/left/image_flipped" />

    <!-- <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." /> -->
    <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
    <arg name="queue_size" default="3" doc="Specigy queue_size of input image subscribers" />

    <arg name="histogram" default="[25, 255, 10, 15, 25, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0, 0, 0, 10]" doc="Histogram of tracked color object" />
    <arg name="vmin" default="10" doc="Min threshould of lightness." />
    <arg name="vmax" default="255" doc="Max threshould of lightness." />
    <arg name="smin" default="60" doc="Min value of saturation." />
  </include>

    <!-- launch video stream -->
    <include file="$(find opencv_apps)/launch/camshift.launch">
      <arg name="node_name" default="featurization_right" />
      <arg name="image" value="/stereo/right/image_flipped" />
  
      <!-- <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." /> -->
      <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
      <arg name="queue_size" default="3" doc="Specigy queue_size of input image subscribers" />
  
      <arg name="histogram" default="[25, 255, 10, 15, 25, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0, 0, 0, 10]" doc="Histogram of tracked color object" />
      <arg name="vmin" default="10" doc="Min threshould of lightness." />
      <arg name="vmax" default="255" doc="Max threshould of lightness." />
      <arg name="smin" default="60" doc="Min value of saturation." />
    </include>

</launch>

 